doc/sphinx/changes.rst

```diff
@@ -8,6 +8,753 @@ Recent changes
+  :g:`Ropp_pos`,
+  :g:`Rplus_0_l_uniq`,
+  :g:`Rplus_eq_0`,
+  :g:`Rplus_ge_reg_r`,
+  :g:`Rplus_gt_reg_r`,
+  :g:`Rplus_minus_assoc`,
+  :g:`Rplus_minus_l`,
+  :g:`Rplus_minus_r`,
+  :g:`Rplus_minus_swap`,
+  :g:`Rplus_neg_lt`,
+  :g:`Rplus_neg_neg`,
+  :g:`Rplus_neg_npos`,
+  :g:`Rplus_nneg_ge`,
+  :g:`Rplus_nneg_nneg`,
+  :g:`Rplus_nneg_pos`,
+  :g:`Rplus_npos_le`,
+  :g:`Rplus_npos_neg`,
+  :g:`Rplus_npos_npos`,
+  :g:`Rplus_pos_gt`,
+  :g:`Rplus_pos_nneg`,
+  :g:`Rplus_pos_pos`,
+  :g:`Rsqr_def`
+
+  lemmas in `Reals/R_Ifp.v`:
+  :g:`Int_part_spec`,
+  :g:`Rplus_Int_part_frac_part`,
+  :g:`Int_part_frac_part_spec`
+
+  (`#17036 <https://github.com/coq/coq/pull/17036>`_,
+  by Pierre Rousselin, reviewer Laurent Théry).
+
+- **Changed**
+  lemmas in `Reals/RIneq.v`
+
+  - :g:`completeness_weak` renamed as :g:`upper_bound_thm`,
+  - :g:`le_epsilon` renamed as :g:`Rle_epsilon`,
+  - :g:`Rplus_eq_R0` renamed as :g:`Rplus_eq_0`,
+  - :g:`Req_EM_T` renamed as :g:`Req_dec_T`,
+  - :g:`Rinv_r_simpl_m` renamed as :g:`Rmult_inv_r_id_m`,
+  - :g:`Rinv_r_simpl_l` renamed as :g:`Rmult_inv_r_id_l`,
+  - :g:`Rinv_r_simpl_r` renamed as :g:`Rmult_inv_m_id_r`,
+  - :g:`tech_Rgt_minus` renamed as :g:`Rgt_minus_pos`,
+  - :g:`tech_Rplus` renamed as :g:`Rplus_le_lt_0_neq_0`,
+  - :g:`IZR_POS_xI` modified with `2` instead of `1 + 1`,
+  - :g:`IZR_POS_xO` modified with `2` instead of `1 + 1`,
+  - :g:`Rge_refl` modified with `>=` instead of `<=`
+
+  (`#17036 <https://github.com/coq/coq/pull/17036>`_,
+  by Pierre Rousselin, reviewer Laurent Théry).
+
+- **Deprecated**
+  lemmas in `Reals/RIneq.v`
+
+  :g:`inser_trans_R`,
+  :g:`IZR_neq`,
+  :g:`double`,
+  :g:`double_var`,
+  :g:`Rinv_mult_simpl`,
+  :g:`Rle_Rinv`,
+  :g:`Rlt_Rminus`,
+  :g:`Rminus_eq_0`,
+  :g:`Rminus_gt_0_lt`,
+  :g:`Ropp_div`,
+  :g:`Ropp_minus_distr'`,
+  :g:`Rplus_sqr_eq_0_l`,
+  :g:`sum_inequa_Rle_lt_depr`,
+  :g:`S_O_plus_INR_depr`,
+  :g:`single_z_r_R1_depr`,
+  :g:`tech_single_z_r_R1_depr`,
+
+  (`#17036 <https://github.com/coq/coq/pull/17036>`_,
+  by Pierre Rousselin, reviewer Laurent Théry).
+- **Added:** lemmas :g:`concat_length`, :g:`flat_map_length`,
+  :g:`flat_map_constant_length`, :g:`list_power_length` to `Lists.List`
+  (`#17082 <https://github.com/coq/coq/pull/17082>`_,
+  by Stefan Haan with help from Olivier Laurent).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Sphinx 4.5.0 or above is now required to build the reference manual, so now /
+  can be used as a quick search shortcut and Esc as a shortcut to remove search
+  highlighting
+  (`#17772 <https://github.com/coq/coq/pull/17772>`_,
+  fixes `#15778 <https://github.com/coq/coq/issues/15778>`_,
+  by Ana Borges).
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- **Added:**
+  a hook in the coercion mechanism to enable programming coercions in
+  external metalanguages such as Ltac, Ltac2, Elpi or OCaml plugins
+  (`#17794 <https://github.com/coq/coq/pull/17794>`_,
+  by Pierre Roux).
```

Comment:
Wouldn't this be appropriate under the "Specification language, type inference" section?

Reply:
@proux01 Where do you want it?

Reply:
You're right it's probably better under "Specification language, type inference", feel free to move it.